explore/cpp/utility.h

```diff
@@ -232,7 +232,7 @@ class NumberUtils
 class StringUtils
 {
 public:
-  static inline string to_string(size_t& n)
+  static inline string to_string(size_t n)
```

Comment:
In cbify.cc I’m calling to_string(<temp result>) and one can’t take &<temp result>

Also I don’t see any benefit of using the reference here…